### PR TITLE
Fix: removing overflow hidden

### PR DESCRIPTION
**Pull request by mirayavandiepen**
### Summary ⤵️
The code `<body>` and `<html>` get added the same styling attributes once scroll-lock is enabled.

Style-Attribute: `overscroll-behavior:none!important;-webkit-overflow-scrolling:auto!important;overflow:hidden!important;`

Once you remove `overflow:hidden!important;` on the body, all the sticky elements will not disappear anymore, and the scroll-lock still works. However, we can not remove it on both `<body>` and `<HTML>` because if both get removed, the sticky element will remain visible, **but the scroll-lock won't work**. In this case, I've only removed it from the `<body>.` By the way, I've also tested it on our site with the **HTML Inspector**.

### Conclusion 💡
 We need two different variables for the two different styling attributes on the` <HTML>` and `<body>,` where one contains the `overflow:hidden!important;` and the other doesn't. Moreover, the sticky-elements will stay visible, and at the same time, the scroll lock will work.

### CODE REVIEW

File: src/lib.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ const elementDatasetName = "tsslockid";
 const styleBackupDatasetName = "tsslockstyle";
 const version = '0.11.3';
 
-const lockStyle = ";overscroll-behavior:none!important;-webkit-overflow-scrolling: auto!important;overflow:hidden!important;";
+const lockStyleHTML = ";overscroll-behavior:none!important;-webkit-overflow-scrolling: auto!important;overflow:hidden!important;";
```
Comment (ttristan) [inline src/lib.ts]:
```suggestion
const lockStyleBody = ";overscroll-behavior:none!important;-webkit-overflow-scrolling: auto!important;";
const lockStyleHTML = lockStyleBody + "overflow:hidden!important;";
```

File: src/lib.ts [hunk 2]
```diff
@@ -6,7 +6,8 @@ const elementDatasetName = "tsslockid";
...
+const lockStyleBody = ";overscroll-behavior:none!important;-webkit-overflow-scrolling: auto!important;";
```
Comment (ttristan) [inline src/lib.ts:9]:
```suggestion
```

Comment (mirayavandiepen) [reply src/lib.ts:9]: Did it manually

**Comment by ttristan** [comment]
Hi Miraya, thanks for the PR. The CTA Example you give here is a bit out of context here on github. But since we talked I know that you mean sticky elements on the page disappearing when the scroll lock is active.

We should give this some time for testing. 

Until then: Please refer to [previous PRs](https://github.com/ttristan/ts-body-scroll-lock/pulls?q=is%3Apr+is%3Aclosed+) and make sure to also check if build files buy running `npm run build:all` script and also bump the version to a new minor (0.12.0)
Thank you!

**Comment by k-ruben** [comment]
Good catch 👍 
An example would be awesome 🔥

**Comment by mirayavandiepen** [comment]
> Good catch +1 An example would be awesome fire

All done, an example should already be included 💯

**Pull request merged**